test/controller/pokemon.js

```diff
@@ -66,24 +66,28 @@ describe('PokemonController', () => {
       const res = await otherAgent.get(`/p/${publicId}`);
       expect(res.statusCode).to.equal(200);
       expect(res.body.pid).to.exist;
+      expect(res.body.speciesName).to.exist;
```

Comment:
Could you add a check to check what the species name should actually be as well? Just to make sure it's not just copying over the id or something like that :-)